### PR TITLE
treewide: rename ZyXEL to Zyxel

### PR DESCRIPTION
**Pull request by GoetzGoerisch**
The company Zyxel rebranded some years ago.

Currently the casing is according to the old branding even for newer devices which already use the new branding.
This commit aligns the casing of Zyxel everywhere.

As per discussion with @svanheule I posting this PR to receive feedback from the developer community.

Signed-off-by: Goetz Goerisch <ggoerisch@gmail.com>

### CODE REVIEW

**Comment by Borromini** [comment]
Oh well. I liked the quirky spelling.

**Comment by GoetzGoerisch** [comment]
Have already correct my last commit, was an oversight on my end.
Currently waiting for the CI to complete before I push again.

**Comment by GoetzGoerisch** [comment]
> Oh well. I liked the quirky spelling.

Thank you for your feedback.

**Comment by Neustradamus** [comment]
Okay to use the new name

**Review by chunkeey — CHANGES_REQUESTED** [review]
Please don't modify these properties.

Reason is that these are fixed Identifiers that are being used by a case-sensitive memcmp in the kernel.

https://elixir.bootlin.com/linux/latest/source/drivers/net/wireless/ath/ath10k/core.c#L1367

EDIT: This was stuck in "pending" for a while?!

File: target/linux/ipq40xx/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq4018-nbg6617.dts [hunk 1]
```diff
@@ -311,12 +311,12 @@
 
 &wifi0 {
 	status = "okay";
-	qcom,ath10k-calibration-variant = "ZyXEL-NBG6617";
+	qcom,ath10k-calibration-variant = "Zyxel-NBG6617";
```
[the remark this answers is not in the document]
Comment (GoetzGoerisch) [reply target/linux/ipq40xx/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq4018-nbg6617.dts:314]: This proposal would need to be upstreamed as well. This PR was mainly to gather feedback.
Another way would be to recase it only where it is visible to the user in UI or menuconfig?

[the remark this answers is not in the document]
Comment (GoetzGoerisch) [reply target/linux/ipq40xx/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq4018-nbg6617.dts]: Reverted this and a few other places. Thank you for your review.

**Comment by GoetzGoerisch** [comment]
@chunkeey please have a look, when time permits.

File: README.md
Comment (hauke) [inline README.md:1]: Please do this in a separate PR.

File: target/linux/kirkwood/patches-6.1/107-03-nsa325.patch [hunk 2]
```diff
@@ -1,7 +1,11 @@
 --- a/arch/arm/boot/dts/kirkwood-nsa325.dts
 +++ b/arch/arm/boot/dts/kirkwood-nsa325.dts
-@@ -15,6 +15,13 @@
- 	model = "Zyxel NSA325";
+@@ -12,9 +12,16 @@
+ #include "kirkwood-nsa3x0-common.dtsi"
+ 
+ / {
+-	model = "ZyXEL NSA325";
++	model = "Zyxel NSA325";
```
Comment (hauke) [inline target/linux/kirkwood/patches-6.1/107-03-nsa325.patch:8]: Please do not update this patch, just fix this in the upstream Linux kernel.

**Comment by Neustradamus** [comment]
@GoetzGoerisch: Any progress on this PR?

**Comment by GoetzGoerisch** [comment]
@Neustradamus Haven't had any time the last week. Will take up the next days.


@hauke how to upstream this patch? Have no experience in contributing to the kernel?

**Comment by hauke** [comment]
This is the official guide for contribution to the Linux kernel: https://docs.kernel.org/process/submitting-patches.html
There are also some other shorter guides in the internet.

**Comment by GoetzGoerisch** [comment]
Thank you @hauke.
For now I have updated this PR per your review. Hope this is ok to be merged.

File: target/linux/ramips/image/mt7620.mk [hunk 3]
```diff
@@ -1574,51 +1574,51 @@ TARGET_DEVICES += zte_q7
...
   IMAGE_SIZE := 16064k
-  DEVICE_VENDOR := ZyXEL
+  DEVICE_VENDOR := Zyxel
   DEVICE_MODEL := Keenetic Viva
   DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport \
 	kmod-switch-rtl8367b
   IMAGES += factory.bin
   IMAGE/factory.bin := $$(sysupgrade_bin) | pad-to 64k | check-size | \
-	zyimage -d 8997 -v "ZyXEL Keenetic Viva"
+	zyimage -d 8997 -v "Zyxel Keenetic Viva"
```
Comment (hauke) [inline target/linux/ramips/image/mt7620.mk:1621]: This `ZyXEL` is going into the firmware image header.
Maybe this is checked by the bootloader or the vendor firmware when updating.

I would prefer if you do not change the writing in the commands to `zyimage` if you or someone else haven"t tested such an image.

Comment (GoetzGoerisch) [reply target/linux/ramips/image/mt7620.mk]: Reverted this and the other four occurences.
Thanks for the review.
